2018-09-17 14:13:42,843 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 14:13:42,859 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 14:13:42,859 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 14:13:42,859 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-17 14:13:59,218 [11] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/16 10:12:07] | Kind: subscription_charged_successfully
2018-09-17 14:14:20,578 [11] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:7ea4d356-7546-4c41-8e9f-633477cb295c
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-17 14:14:28,140 [11] ERROR SkieerWeb - Process Error.SubscriptionId:38c4ae17-f854-44af-8ec5-4590af06ed32,PaymentNo:hpqne2th
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-17 15:13:58,906 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 15:13:58,906 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 15:13:58,906 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 15:13:58,906 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-17 15:14:01,046 [11] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/16 10:12:07] | Kind: subscription_charged_successfully
2018-09-17 15:14:22,171 [11] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:7ea4d356-7546-4c41-8e9f-633477cb295c
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-17 15:14:24,843 [11] ERROR SkieerWeb - Process Error.SubscriptionId:38c4ae17-f854-44af-8ec5-4590af06ed32,PaymentNo:hpqne2th
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-17 16:14:29,234 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 16:14:29,250 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 16:14:29,250 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 16:14:29,250 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-17 16:14:30,921 [11] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/16 10:12:07] | Kind: subscription_charged_successfully
2018-09-17 16:14:52,015 [11] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:7ea4d356-7546-4c41-8e9f-633477cb295c
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-17 16:14:53,109 [11] ERROR SkieerWeb - Process Error.SubscriptionId:38c4ae17-f854-44af-8ec5-4590af06ed32,PaymentNo:hpqne2th
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-17 17:15:03,906 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 17:15:03,906 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 17:15:03,906 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 17:15:03,906 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-17 17:15:05,703 [10] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/16 10:12:07] | Kind: subscription_charged_successfully
2018-09-17 17:15:26,812 [10] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:7ea4d356-7546-4c41-8e9f-633477cb295c
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-17 17:15:27,875 [10] ERROR SkieerWeb - Process Error.SubscriptionId:38c4ae17-f854-44af-8ec5-4590af06ed32,PaymentNo:hpqne2th
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-17 18:15:31,437 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 18:15:31,453 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 18:15:31,453 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 18:15:31,453 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-17 18:15:33,218 [10] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/16 10:12:07] | Kind: subscription_charged_successfully
2018-09-17 18:15:54,359 [10] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:7ea4d356-7546-4c41-8e9f-633477cb295c
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-17 18:15:55,437 [10] ERROR SkieerWeb - Process Error.SubscriptionId:38c4ae17-f854-44af-8ec5-4590af06ed32,PaymentNo:hpqne2th
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-17 20:21:06,062 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 20:21:06,062 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 20:21:06,062 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 20:21:06,062 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-17 22:20:27,171 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-17 22:20:27,171 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-17 22:20:27,171 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-17 22:20:27,171 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
